mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -652,14 +652,16 @@ def _get_attributes_filtering_clauses(parsed):
         key_type = sql_statement.get('type')
         key_name = sql_statement.get('key')
         value = sql_statement.get('value')
-        comparator = sql_statement.get('comparator')
+        comparator = sql_statement.get('comparator').lower()
         if SearchUtils.is_attribute(key_type, comparator):
-            # validity of the comparator is checked in SearchUtils.parse_search_filter()
-            op = SearchUtils.filter_ops.get(comparator)
-            if op:
-                # key_name is guaranteed to be a valid searchable attribute of entities.RunInfo
-                # by the call to parse_search_filter
-                attribute_name = SqlRun.get_attribute_name(key_name)
+            # key_name is guaranteed to be a valid searchable attribute of entities.RunInfo
+            # by the call to parse_search_filter
+            attribute_name = SqlRun.get_attribute_name(key_name)
+            if comparator in ['like', 'ilike']:
+                op = SearchUtils.get_sql_filter_ops(getattr(SqlRun, attribute_name), comparator)
+                clauses.append(op(value))
+            elif comparator in SearchUtils.filter_ops:
+                op = SearchUtils.filter_ops.get(comparator)
```

Comment:
Can these two cases share `getattr(SqlRun, attribute_name)`. 
Also add a constant like `CASE_INSENSITIVE_STRING_COMPARISON_OPERATORS` inside `SearchUtils` as `set(['LIKE', 'ILIKE'])` and use that everywhere.

Reply:
Done